_modules/sesceph/__init__.py

```diff
@@ -161,14 +162,22 @@ def _update_partition(action, dev, description):
 
 
 
-def zap(dev):
+def zap(dev = None, **kwargs):
     """
     Destroy the partition table and content of a given disk.
     """
+    if dev is not None:
+        log.warning("Depricated use of function function use kwargs")
+    test_dev = kwargs.get("dev", None)
+    if test_dev != None:
+        dev = test_dev
```

Comment:
What about "dev = kwargs.get("dev", dev)?

On Mon, Apr 4, 2016 at 4:24 AM, Owen Synge <notifications@github.com> wrote:

> In _modules/sesceph/__init__.py
> <https://github.com/oms4suse/sesceph/pull/22#discussion_r58341840>:
>
> >      """
> >      Destroy the partition table and content of a given disk.
> >      """
> > +    if dev is not None:
> > +        log.warning("Depricated use of function function use kwargs")
> > +    test_dev = kwargs.get("dev", None)
> > +    if test_dev != None:
> > +        dev = test_dev
>
> no as then dev value as a parameter would be lost if this valuer is set.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/oms4suse/sesceph/pull/22/files/84a48a3b3cd0c85bb689caab139fe6425fd7b5a8#r58341840>
>